junifer/data/masks.py

```diff
@@ -37,13 +38,21 @@
 _masks_path = Path(__file__).parent / "masks"
 
 
-def _fetch_icbm152_brain_gm_mask(target_img: "Nifti1Image", **kwargs):
+def _fetch_icbm152_brain_gm_mask(
+    target_img: "Nifti1Image",
+    extra_data: Optional[Dict[str, Any]] = None,
+    **kwargs,
+):
     """Fetch ICBM152 brain mask and resample.
 
     Parameters
     ----------
     target_img : nibabel.Nifti1Image
         The image to which the mask will be resampled.
+    extra_data : dict, optional
+        The other fields in the data object. Useful for accessing other data
+        kinds that needs to be used in the computation of masks. Default is
+        None. This parameter is not used in this function.
```

Comment:
- ... Default is None ... => ... (default None).
 - Why have the `extra_dict` parameter if it's not used?

Reply:
it's used.

Reply:
sorry, not used there, it was left by mistake.